packages/legacy/core/App/screens/Home.tsx

```diff
@@ -139,11 +148,11 @@ const Home: React.FC<HomeProps> = () => {
             style={{
               paddingHorizontal: 20,
               paddingTop: index === 0 ? 20 : 0,
-              paddingBottom: index === notifications.length - 1 ? 20 : 10,
+              paddingBottom: !showNotifications ? 0 : index === notifications.length - 1 ? 20 : 10,
               backgroundColor: ColorPallet.brand.secondaryBackground,
             }}
           >
-            {DisplayListItemType(item)}
+            { showNotifications && DisplayListItemType(item) }
```

Comment:
`showNotifications` can be removed unless it becomes changeable / configurable. If it does become configurable, then it should simply not render these list items at all. No empty `View`s with padding and background needed

Reply:
Removed the chevron icons and the `showNotification` state, will add them back in next story with the implementations